src/hclib-runtime.c

```diff
@@ -929,17 +932,13 @@ static void _finish_ctx_resume(void *arg) {
     HASSERT(0);
 }
 
-void crt_work_loop(LiteCtx *ctx);
-
 // Based on _help_finish_ctx
 void _help_wait(LiteCtx *ctx) {
     hclib_future_t **continuation_deps = ctx->arg;
     LiteCtx *wait_ctx = ctx->prev;
 
-    hclib_dependent_task_t *task = (hclib_dependent_task_t *)malloc(sizeof(
-                                       hclib_dependent_task_t));
+    hclib_dependent_task_t *task = calloc(1, sizeof(*task));
```

Comment:
I think this was copied over from a C++ source file. You don't need to cast the result of `malloc` in C. Also, using `calloc` instead lets us avoid the separate `memset` call.